fenix/app/src/main/res/values/strings.xml

```diff
@@ -377,7 +377,8 @@
     <string name="preferences_https_only_title">HTTPS-Only Mode</string>
 
     <!-- Preference for removing cookie/consent banners from sites automatically. See reduce_cookie_banner_summary for additional context. -->
-    <string name="preferences_cookie_banner_reduction">Cookie Banner Reduction</string>
+    <string name="preferences_cookie_banner_reduction" moz:RemovedIn="114" tools:ignore="UnusedResources">Cookie Banner Reduction</string>
+    <string name="preferences_cookie_banner_reduction_2">Cookie banner reduction</string>
```

Comment:
@iorgamgabriel I'm still see the previous version on [Figma](https://www.figma.com/file/FAZxkK9IRLlCqyhXBqId7e/Auto-Cookie-Banner-Consent-Desktop-%26-Mobile?node-id=3932-37050&t=zahv53CveVelJt6P-0) could you help me to clarify which is the right one?
<img width="244" alt="image" src="https://user-images.githubusercontent.com/773158/235751371-a5892612-600f-4dcc-9fb1-384f7133c968.png">

Reply:
Chatted with Phil, this is the right change!

Reply:
<img width="448" alt="Screenshot 2023-05-03 at 10 06 55" src="https://user-images.githubusercontent.com/8118371/235851745-d0753d70-f705-4e44-bd32-fc28ddbfae8b.png">

Reply:
This is from [Figma](https://www.figma.com/file/FAZxkK9IRLlCqyhXBqId7e/Auto-Cookie-Banner-Consent-Desktop-%26-Mobile?node-id=3932-37050&t=JMGSEUe3gsRvCrLX-0)